src/components/Tool/Home/Home.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ThemeProvider } from '@material-ui/core';
+import TestRouter from '../../../../__tests__/util/TestRouter';
+import theme from '../../../theme';
+import Home from './Home';
+
+const accountListId = 'account-list-1';
+
+const router = {
+  query: { accountListId },
+  isReady: true,
+};
+
+describe('ToolHome', () => {
+  it('default', () => {
+    const { getByTestId } = render(
+      <ThemeProvider theme={theme}>
+        <TestRouter router={router}>
+          <Home />
+        </TestRouter>
+      </ThemeProvider>,
+    );
+
+    expect(getByTestId('Home')).toBeInTheDocument();
```

Comment:
This isn't much of a test but at least it's checking that the component didn't throw an error. Idk that the home screen is all that prone to bugs so idk that it's worth spending more time on.